imports/state/epics/unverified-warning.js

```diff
@@ -0,0 +1,28 @@
+// import { Accounts } from 'meteor/accounts-base'
+import { Meteor } from 'meteor/meteor'
+import { Subject } from 'rxjs/Subject'
+import { RESEND_VERIFICATION, RESEND_ERROR, RESEND_SUCCESS } from '../../ui/unverified-warning/unverified-warning.actions'
+
+import 'rxjs/add/operator/switchMap'
+import 'rxjs/add/operator/filter'
+
+export const unverifiedWarning = (action$, store) =>
+  action$.ofType(RESEND_VERIFICATION)
+    .filter(() => store.getState().resendVerificationState)
```

Comment:
I don't think it is necessary or makes a difference... `store.getState().resendVerificationState` will always be defined